docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+      - .:/var/www/firefly-iii
```

Comment:
Important note here, since the folder you are mounting doesn't have the _vendor_ folder, you need to run `compose install` separately to make this work.